### PR TITLE
measure: some tweaks

### PR DESCRIPTION
**Pull request by poettering**
A follow-up for #24458.

### CODE REVIEW

File: src/boot/measure.c [hunk 1]
```diff
@@ -62,27 +62,29 @@ static int help(int argc, char *argv[], void *userdata) {
...
-               "     --pcrpkey=PATH      Path to public key for PCR signatures in DER format\n"
                "  -c --current           Use current PCR values\n"
                "     --bank=DIGEST       Select TPM bank (SHA1, SHA256)\n"
                "     --tpm2-device=PATH  Use specified TPM2 device\n"
                "     --private-key=KEY   Private key (PEM) to sign with\n"
                "     --public-key=KEY    Public key (PEM) to validate against\n"
                "     --json=MODE         Output as JSON\n"
                "  -j                     Same as --json=pretty on tty, --json=short otherwise\n"
+               "\n%3$sPE Section Options:%4$s                                             %3$sPE Section%4$s\n"
+               "     --linux=PATH        Path to Linux kernel ELF image file    %7$s .linux\n"
```
Comment (medhefgo) [inline src/boot/measure.c]: If you use the stub the kernel *cannot* be a ELF image…

Comment (poettering) [reply src/boot/measure.c]: hmm, well internally it is an ELF image. But you are right, might be better finding another terminology for that.

Comment (medhefgo) [reply src/boot/measure.c]: Just saying linux kernel image should do

**Comment by poettering** [comment]
Force pushed a new version, with the "ELF" dropped, as @medhefgo suggested. PTAL

**Review by bluca — APPROVED** [review]

**Pull request merged**